Runner.php

```diff
@@ -4,6 +4,8 @@
 
 class Runner
 {
+    const DEFAULT_EXTENSIONS = array("php", "inc", "module");
```

Comment:
This might be excessive, but: Are these default extensions set anywhere inside of phpcodesniffer, and is there a way we can get them out of there?